File: Strings/3358/3358.c
#include <stdio.h>
#include <string.h>

int main(){
    int n, count = 0;
    char x[42];
    scanf("%d", &n);
    for (int i = 0; i<n; i++){
        scanf("%s", &x);

        for (int j = 0; j < strlen(x); j++){
            if (x[j]!= 'a' && x[j]!= 'e' && x[j]!= 'i' && x[j]!= 'o' && x[j]!= 'u' && x[j]!= 'A' && x[j]!= 'E' && x[j]!= 'I' && x[j]!= 'O' && x[j]!= 'U'){
                count++;
                if (count == 3){break;}
            }
            else {
                count = 0;
            }
        }
        if (count == 3){
            printf("%s nao eh facil\n", x);
        }
        else {
            printf("%s eh facil\n", x);
        }
        count = 0;
    }
    return 0;
}

File: Strings/3358/3358.py
n =int(input())
count = 0
for i in range(n):
    y = input()
    x = list(y)
    for j in range(len(x)):
        if x[j]!= 'a' and x[j]!= 'e' and x[j]!= 'i' and x[j]!= 'o' and x[j]!= 'u' and x[j]!= 'A' and x[j]!= 'E' and x[j]!= 'I' and x[j]!= 'O' and x[j]!= 'U':
            count+=1
            if (count == 3):
                break
        else:
            count = 0
    if count == 3:
        print(f"{y} nao eh facil")
    else:
        print(f"{y} eh facil")
    count = 0